docs/src/index.rst

```diff
@@ -1,18 +1,22 @@
 iris-esmf-regrid
 ================
 
-A collection of structured and unstructured ESMF_ regridding schemes for Iris_.
+Iris-esmf-regrid is a package which aims to bridge the gap between Iris_ data
+handling and ESMF_ regridding. This is done largely by way of providing
+Iris_-like regridding schemes.
 
-This rendered documentation is so far limited to iris-esmf-regrid's API. Some
-basic examples and change logs can be found on GitHub_.
```

Comment:
If there is no clear 'division of labour' between the full documentation and the `README`, I suggest migrating everything over to the full documentation. We don't want a situation where both need to be kept up-to-date in tandem.

We can leave anything in the `README` that doesn't have a defined place elsewhere.